ruby/README.md

```diff
@@ -93,6 +109,12 @@ From the project root (rather than the `ruby` directory):
 $ bazel test //ruby/tests/...
 ```
 
+To run tests against the FFI implementation:
+
+```
+$ bazel test //ruby/tests/... //ruby:ffi=enabled --test_env=PROTOCOL_BUFFERS_RUBY_IMPLEMENTATION=FFI
```

Comment:
Why do we need to specify both?  Couldn't we make the `protobuf` target set this env flag based on the config setting?

Reply:
We could change that :), but fair enough

Reply:
Does ruby_library allow it?

Reply:
No - https://github.com/bazelruby/rules_ruby#ruby_library